packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -523,6 +523,7 @@
     "hosts",
     "is_default",
     "is_default_monitoring",
+    "is_internal",
```

Comment:
tbh not sure how to test/check this change, just following the pattern for other config keys

Reply:
You manually modified this file right?

This check was added to make sure that any mapping addition is properly done, by adding a new model version. And I don't see that being done in this PR (looking at `x-pack/plugins/fleet/server/saved_objects/index.ts`)

Please follow the guidelines available in [the model version documentation](https://github.com/elastic/kibana/blob/cadabe5be9f483c6c01ffd4da49ca6bba101b6b1/packages/core/saved-objects/core-saved-objects-server/docs/model_versions.md)

(or follow what was already done for this SO type: https://github.com/elastic/kibana/blob/70508b957a4860b12fd3a2bb9612cfb8a95092e9/x-pack/plugins/fleet/server/saved_objects/index.ts#L312-L321)

Reply:
thanks for the pointer, will check and follow the guidelines!

Reply:
@pgayvallet added a new model version to reflect the addition of the `is_internal` mapping. Btw running `node scripts/check_mappings_update --fix` made quite a lot of changes to the current_mappings.json which are not related to the is_internal, so I ended up still adding 
```
      "is_internal": {
        "type": "boolean",
        "index": false
      },
```
manually (`node scripts/check_mappings_update` passes after I added the model version)